### DISCUSSION TITLE
better selecting of notes in the editor

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#21107] [Q&A] [answered]
https://user-images.githubusercontent.com/78441708/199801766-d664f202-34d9-41ad-92fd-8bccdbf2000d.mp4

before [2022.1031.0](https://github.com/ppy/osu/releases/tag/2022.1031.0), when you tried to drag a selected portion, it dragged it. now, if there are objects behind that selected portion where you click to drag, it selects and drags the objects behind it instead (prioritizing objects with lowest timestamp). if you are trying to drag overlapping items it can be very difficult.

**Comment by Axilotl17** [comment, marked as answer]
i suspect this is related to #20965

**Reply by smoogipoo** [reply]
Definitely feels bad. Not sure if it's related to that issue, but https://github.com/ppy/osu/issues/21114
